apps/prairielearn/src/sprocs/assessment_instances_update.sql

```diff
@@ -33,7 +33,7 @@ BEGIN
     SELECT
         c.id,      g.id,   u.user_id,            a.id,          a.type,
         a.max_points,          ai.max_points,
-        COALESCE(a.max_bonus_points, 0),    ai.max_bonus_points,
+        GREATEST(a.max_bonus_points, 0),    ai.max_bonus_points,
```

Comment:
Is this change simply to guard against the fact that a user might set `assessment.max_bonus_points` to a negative value?

Reply:
Indeed, though this `GREATEST` logic was already in place in the removed line below, I just moved the logic to the query instead.

Reply:
Yep, seems reasonable.